inst/odin/model-targeted-vax.R

```diff
@@ -193,16 +193,86 @@ dim(adults_dose1_denom) <- c(n_group)
 adults_dose2_denom[] <- (S[i, 3] + Ea[i, 3] + Eb[i, 3] + R[i, 3]) * give_dose2_adults[i]
 dim(adults_dose2_denom) <- c(n_group)
 
-### allocate to S
+## decide how many will go to each different state 
 
 ## children 1st doses
+
+children_dose_state_prob[1] <- if (sum(children_dose1_denom[])==0) 0 else
+  S[i,2] * give_dose1_children[i]/sum(children_dose1_denom[])
+children_dose_state_prob[2] <- if (sum(children_dose1_denom[])==0) 0 else
+  Ea[i,2] * give_dose1_children[i]/sum(children_dose1_denom[]) 
+children_dose_state_prob[3] <- if (sum(children_dose1_denom[])==0) 0 else
+  Eb[i,2] * give_dose1_children[i]/sum(children_dose1_denom[])
+children_dose_state_prob[4] <- if (sum(children_dose1_denom[])==0) 0 else
+  R[i,2] * give_dose1_children[i]/sum(children_dose1_denom[])
+dim(children_dose_state_prob) <- 4
```

Comment:
I would include age here
```suggestion
children_dose_state_prob[i, 1] <- if (sum(children_dose1_denom)==0) 0 else
  S[i, 2] * give_dose1_children[i] / sum(children_dose1_denom)
children_dose_state_prob[i, 2] <- if (sum(children_dose1_denom)==0) 0 else
  Ea[i, 2] * give_dose1_children[i] / sum(children_dose1_denom) 
children_dose_state_prob[i, 3] <- if (sum(children_dose1_denom)==0) 0 else
  Eb[i, 2] * give_dose1_children[i] / sum(children_dose1_denom)
children_dose_state_prob[i, 4] <- if (sum(children_dose1_denom)==0) 0 else
  R[i, 2] * give_dose1_children[i] /sum(children_dose1_denom)
```